### DISCUSSION TITLE
GUI Creator

### DISCUSSION THREAD

**Discussion by Callial** [Zrips/CMI#2944] [Feature Requests]
**Description of feature/improvement request:**
Custom GUI creator with permission system for buttons and/or pages
---
### Detailed explanation of what it should do, when and in what situation: ###

It would be nice to be able to create custom chest or book GUI's to run commands and custom aliases. I would like to be able to have certain buttons locked and/or hidden if players don't have the permission for them. A built-in page system would be nice but not necessary 

### Basic explanation as to why this feature is needed and/or useful to have: ###

Server owners need to monetize their servers. By giving them a custom GUI creator they can cut out one more plugin from their list, as well as allowing them to create things like custom chat suffixes and prefixes using only a permission plugin and CMI without having to create hundreds of custom aliases.

**Comment by hamcoremusic** [comment]
I would love to see this so I can make a custom GUI for new players to pick servers.
